README.md

```diff
@@ -217,5 +217,5 @@ make help
 ```
 
 # License
-
+git@github.com:prgcont/workshop-ASB.git
```

Comment:
Is this line intended to be here?

Reply:
ups, little advertisement can't hurt they say. 
Sorry for that. 
It's gone now.